### PR TITLE
SCMOD-3298: Set dropwizard version to 1.3.2

### PR DESCRIPTION
**Pull request by rado-straka**
https://jira.autonomy.com/browse/SCMOD-3298

### CODE REVIEW

**Comment by dermot-hardy** [comment]
Is there a developer build for this?  Has it been dev tested?

**Review by dermot-hardy — CHANGES_REQUESTED** [review]
Can this be done using the BOM the same way the WorkerFramework is done:
https://github.com/WorkerFramework/worker-framework/pull/66

**Comment by rado-straka** [comment]
Updated

File: release-notes-1.3.0.md [hunk 1]
```diff
@@ -6,5 +6,7 @@ ${version-number}
 #### New Features
  - [SCMOD-3574](https://jira.autonomy.com/browse/SCMOD-3574): Marathon client version update  
     The version of the Marathon client used has been updated to support using Vault for secret management.
+ - [SCMOD-4072](https://jira.autonomy.com/browse/SCMOD-4072): Updated to use latest version of dropwizard ([1.3.2]
+    (https://github.com/dropwizard/dropwizard/tree/v1.3.2))
```
Comment (dermot-hardy) [inline release-notes-1.3.0.md]: Doesn't look right:
[https://github.com/Autoscaler/autoscaler/blob/8866d84/release-notes-1.3.0.md](https://github.com/Autoscaler/autoscaler/blob/8866d84bfcd403a59fc5a507555751b05c0c0fc8/release-notes-1.3.0.md)

File: autoscale-acceptance-tests/pom.xml [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+            </plugin>
+        </plugins>
+    </build>
+    
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+</project>
```
Comment (dermot-hardy) [inline autoscale-acceptance-tests/pom.xml]: Take a new line

File: autoscale-acceptance-tests/src/integration-test/java/com/hpe/caf/autoscale/tests/AutoscalerIT.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+import java.io.IOException;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClients;
+
+import org.junit.Test;
+
+public class AutoscalerIT
```
Comment (dermot-hardy) [inline autoscale-acceptance-tests/src/integration-test/java/com/hpe/caf/autoscale/tests/AutoscalerIT.java]: Class should be `final`

File: autoscale-acceptance-tests/src/integration-test/java/com/hpe/caf/autoscale/tests/AutoscalerIT.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        assertTrue(healthCheck(swarmRequestUrl).getStatusLine().getStatusCode() == 200);
+    }
+    
+    @Test
+    public void testMarathonHealthCheck() throws IOException {
+        final String marathonRequestUrl = System.getenv("AUTOSCALE_MARATHON_HEALTHCHECK_URL");
+        assertTrue(healthCheck(marathonRequestUrl).getStatusLine().getStatusCode() == 200);
+    }
+    
+    public HttpResponse healthCheck(String requestUrl) throws IOException {
```
Comment (dermot-hardy) [inline autoscale-acceptance-tests/src/integration-test/java/com/hpe/caf/autoscale/tests/AutoscalerIT.java]: Parameter should be `final`.

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**